interface/wx/bmpbndl.h

```diff
@@ -278,12 +279,16 @@ class wxBitmapBundle
         @note Converting text objects to path objects will allow them to be
             rasterized as expected. This can be done in an SVG editor such as
             Inkscape. (In Inkscape, select a text object and choose
-            "Object to Path" from the "Path" menu.)\n
+            "Object to Path" from the "Path" menu.)\n\n
             Converting embedded images to paths from an SVG editor will
             allow them to be rasterized. For example, selecting "Trace Bitmap"
             from the "Path" menu in Inkscape can perform this. This is only
             recommended for simple images, however, as more complex images
-            may not rasterize well.
+            may not rasterize well.\n\n
+            To work around clip paths, you can manually cut and remove portions
```

Comment:
Isn't there something missing here?
```suggestion
            To work around lack of support for clip paths, you can manually cut and remove portions
```